tests/cypress/support/commands_annotations_actions.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
```

Comment:
Should we add in this file a command that fills up action parameters? 
It may be not used in public tests but can be useful in private repo and for testing future public actions.

Reply:
We will, when we need it. 
Not we do not have any actions with parameters, nothing to test now

Reply:
We will, when we need it. 
Not we do not have any actions with parameters (even in private repo), nothing to test now